### DISCUSSION TITLE
Why is %edition% used in configuration instead of widely accepted %comment%?

### DISCUSSION THREAD

**Discussion by Groxxxy** [TT-ReBORN/Georgia-ReBORN#78] [Q&A] [answered]
> **Comment**: Disambiguation Comment - the comment entered to help distinguish one release from another (e.g.: Deluxe version with 2 bonus tracks).

http://picard-docs.musicbrainz.org/en/variables/tags_basic.html

Hi! I respect the work but I am wondering if a vorbis comment field exists for documenting special and deluxe editions why is a totally new field like %edition% being relied upon?

PS. Thanks TT, I have restructured my music library to work with file system view ;)

Thanks!

**Comment by TT-ReBORN** [comment, marked as answer]
Hi,

the %comment% tag is mostly used to write comments about the album as the name suggests. Most artist write their website in the comment field or users write some notes about the album. The %edition% tag is the right tag to use for various edition for the album,
because it also sorts first by year, see in the config file.

If you're using the the latest beta, the library now also respects the sorting first by %original release date% -> %originaldate% -> %date%
That means, you can also use the other library sort options and you're not bound to use only "View by Folder Structure".

-TT
